pkg/sdkserver/localsdk.go

```diff
@@ -197,6 +198,8 @@ func (l *LocalSDKServer) recordRequestWithValue(request string, value string, ob
 			fieldVal = l.gs.ObjectMeta.Uid
 		case "PlayerCapacity":
 			fieldVal = strconv.FormatInt(l.gs.Status.Players.Capacity, 10)
+		case "PlayerIDs":
+			fieldVal = strings.Join(l.gs.Status.Players.IDs, "")
```

Comment:
Do we need to use separator here? This seems to be for test, but might be more readable.

Reply:
Just tried it, and it didn't break any tests! So yep, adding this in, in case people want to test against multiple values in the future.